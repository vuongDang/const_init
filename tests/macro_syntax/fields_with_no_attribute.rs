use const_init::ConstInit;

#[derive(ConstInit)]
struct FooBar {
    #[const_init(value = true)]
    foo: bool,
    bar: usize,
}

fn main() {}
