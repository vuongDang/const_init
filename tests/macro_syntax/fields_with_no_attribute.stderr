error: proc-macro derive panicked
 --> tests/macro_syntax/fields_with_no_attribute.rs:3:10
  |
3 | #[derive(ConstInit)]
  |          ^^^^^^^^^
  |
  = help: message: called `Option::unwrap()` on a `None` value
